src/wazuh_modules/vulnerability_scanner/src/vulnerabilityReport/vulnerabilityReport.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Wazuh vulnerability scanner - Policy Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * November 22, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _VULNERABILITY_REPORT_HPP
+#define _VULNERABILITY_REPORT_HPP
+
+#include "json.hpp"
+#include "socketClient.hpp"
+#include "vulnerabilityScanner.hpp"
+#include <fcntl.h>
+#include <netinet/in.h>
+#include <sys/epoll.h>
+#include <sys/socket.h>
+#include <sys/stat.h>
+#include <sys/un.h>
+#include <unistd.h>
```

Comment:
Maybe these headers are not all necessary. I tested to compile with only these ones:
```
#include "json.hpp"
#include "socketClient.hpp"
```

Reply:
Most of them were unnecessary, yes. I added vulnerabilityScanner.h for module definition.